Update site-recovery-sql.md

jboeshart opened the pull request:
Added notes around Azure to Azure replication scenarios, please confirm I'm correct on these as I wasn't 100% sure on the FCI options. Also fixed a typo and added "2016" in a couple of places.

azurecla commented:
Hi __@jtucker__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jason.tucker). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Ja-Dunn commented:
@jtucker - Thanks for your contribution. We are in the process of refactoring the Azure technical articles, so we can’t merge this PR. When the refactoring is complete, we can give you a link to the new location of the article you updated so you can resubmit your update, if you want to.

ktoliver commented:
@jtucker - Thanks for your contribution to the Azure docs. On November 16th, we announced the availability of our Azure technical documentation in a new location on [docs.microsoft.com](https://docs.microsoft.com). You can [read about it on our team blog](https://docs.microsoft.com/teamblog/docs-november-update). This change means the article source lives in a new GitHub repository. Because there are some changes in the source markdown as a result of the move, we can’t pull your contribution over to the new repo without creating a ton of merge conflicts. We invite you to resubmit the change (if you are willing) to the new version of the article at [https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-diagnostic-how-to-use-elasticsearch](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-diagnostic-how-to-use-elasticsearch). Thanks for your understanding.

jboeshart committed:
Update site-recovery-sql.md

Added notes around Azure to Azure replication scenarios, please confirm I'm correct on these as I wasn't 100% sure on the FCI options. Also fixed a typo and added "2016" in a couple of places.
